src/context_store/orchestrator.py

```
@@ -129,6 +131,64 @@ async def _check_vector_dimension(self) -> None:
+    async def session_flush(
+        self,
+        conversation_log: str,
+        session_id: str | None = None,
+        project: str | None = None,
+        tags: list[str] | None = None,
+    ) -> dict[str, Any]:
+        """会話ログをバックグラウンドでバッチ保存する (Fire-and-forget)。
+
+        Args:
+            conversation_log: 会話ログ全文。
+            session_id: セッション ID（None の場合は自動生成）。
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)